integration/imagetags/Tiltfile

```diff
@@ -4,5 +4,6 @@ docker_build(repo + '/imagetags-common:stretch', 'common-stretch')
 docker_build(repo + '/imagetags-app', 'app')
 docker_build(repo + '/imagetags-app:stretch', 'app-stretch')
 
-k8s_resource("regular", yaml='app.yaml', image=repo + '/imagetags-app', port_forwards='31000:8000')
-k8s_resource("stretch", yaml='app-stretch.yaml', image=repo + '/imagetags-app:stretch', port_forwards='31001:8000')
+k8s_yaml(['app.yaml', 'app-stretch.yaml'])
+k8s_resource('imagetags', new_name='regular', port_forwards='31000:8000')
```

Comment:
it feels like a lot of the code complexity is because we're changing the signature of an existing function

have we considered just using a new function name?

e.g., k8s_workload?

i wish the spec had some "Alternative Considered" section